packages/components/src/components/notification/notification.stories.ts

```diff
@@ -0,0 +1,270 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+import '../../solid-components';
+import { html } from 'lit';
+import { storybookDefaults, storybookHelpers, storybookTemplate } from '../../../scripts/storybook/helper';
+import { userEvent } from '@storybook/testing-library';
+import { waitUntil } from '@open-wc/testing-helpers';
+const { argTypes, parameters } = storybookDefaults('sd-notification');
+const { generateTemplate } = storybookTemplate('sd-notification');
+const { overrideArgs } = storybookHelpers('sd-notification');
+
+export default {
+  title: 'Components/sd-notification',
+  component: 'sd-notification',
+  args: overrideArgs([
+    {
+      type: 'slot',
+      name: 'default',
+      value: `<div class="slot slot--border slot--text h-8 my-auto w-full">Default slot</div>`
+    },
+    {
+      type: 'attribute',
+      name: 'open',
+      value: true
+    }
+  ]),
+  argTypes,
+  parameters: { ...parameters },
+  decorators: [
+    (story: () => typeof html) => html`
+      <style>
+        td.template {
+          width: 100%;
+        }
+      </style>
+      ${story()}
+    `
+  ]
+};
+
+/**
+ * This shows sd-notification in its default state.
+ */
+
+export const Default = {
+  render: (args: any) => {
+    return generateTemplate({
+      args
+    });
+  }
+};
+
+/**
+ * Use the `variant` attribute to change the theme of the notification.
+ */
+
+export const Variants = {
+  parameters: { controls: { exclude: ['variant'] } },
```

Comment:
Since they're always open here, would be good to exclude that too from controls? And maybe check the other stories for same :)